src/SingleGame.jsx

```diff
@@ -48,10 +52,17 @@ function SingleGame() {
           created at: {createDate(singleReview.created_at)}
         </p>
         <p className="single-cat">category: {singleReview.category}</p>
-        <button className="single-button">👎</button>
-        <button className="single-button">👍</button>
+        <button className="single-button" onClick={voteChangeDown}>
+          👎
```

Comment:
I love the use of emojis in your post! 😎 I noticed that the presentation of numbers for upvotes and downvotes might be a bit confusing. To enhance clarity, I'd suggest placing the numbers alongside the respective emojis, allowing users to easily identify which number corresponds to each emoji 👍